student.rb

```diff
@@ -0,0 +1,19 @@
+require './person'
+
+class Student < Person
+  attr_accessor :classroom
+
+  def initialize(age, name = 'unknown', classroom:, parent_permission: true)
+    super(age, name, parent_permission: parent_permission)
+    @classroom = classroom
+  end
+
+  def play_hooky
+    '¯\(ツ)/¯'
+  end
+
+  def add_classroom(classroom)
+    @classroom.students.push(self) unless @classroom.students.include?(self)
+    classroom.students.push(self) unless classroom.students.iclude?(self)
+  end
```

Comment:
- [ ] Kindly note that you did not refactor this method the way it was highlighted. The reviewer asked that you remove only `.student` from `@classroom.student` so that we can initialize the `@classroom`

  ```
  def add_classroom(classroom)
    @classroom = classroom
    classroom.students.push(self) unless classroom.students.include?(self)
  end
  ```